code/web/src/ui/image/Tile.js

```diff
@@ -26,14 +26,13 @@ const Tile = (props) => {
 Tile.propTypes = {
   image: PropTypes.string.isRequired,
   style: PropTypes.object,
-  width: PropTypes.number,
-  height: PropTypes.number,
   shadow: PropTypes.string
```

Comment:
you can't require width and height to be a number and then set its default to a string.